### ISSUE TITLE
[TabStrip] Accessibility issues

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#2306]
Tested browsers:

- Chrome
- Firefox
- Internet Explorer 11

Tested screen readers:

- NVDA
- JAWS

Issues:

When tabbing to the TabStrip, the screen reader will announce "Tab Control", but nothing else to provide context. The TabStrip component has no configuration to provide more context.

When switching between tabs in the TabStrip component using arrow keys, the screen reader will not announce the change.

Example:

The TabStrip component documentation - [ https://www.telerik.com/kendo-angular-ui/components/layout/tabstrip/]( https://www.telerik.com/kendo-angular-ui/components/layout/tabstrip/)

Suggested fix:

The following example features a customized component that behaves as expected according to W3 aria best practices - [https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html)

The solution is not meant to provide the exact implementation, but rather to show what attributes could be added to give the expected behavior. 

[https://stackblitz.com/edit/kendo-tabstrip-accessible](https://stackblitz.com/edit/kendo-tabstrip-accessible)

At a high level, the solution does the following:

- Dynamically sets the current selected tab's tabindex attribute to 0, and sets the tabindex of all other tabs to -1.
- This makes the screen reader announce the tabs properly. Setting all other tabs to tabindex="-1" preserves the current tab flow behavior
- Removes the tab index from the tablist `<ul>` element. This isn't necessary when the selected tab has a tabindex
- Sets focus to the selected tab when the selected tab changes
- This makes the screen reader announce the tab change when using arrow keys

The solution preserves the existing behavior but enhances the experience for sighted users. One difference worth noting is adding focus to the tabs will add extra styling to the tab for the focus state. If this isn't desired it could be removed via css. 

Please take this issue and the provided fix into consideration.

**Internal info: Based on a private support thread - Ticket ID: 1411711**

**Comment by siyka-marinova** [comment]
The issue needs further fixing and improvements. Here are some quick suggestions:
- add role="presentation" attribute to the <ul> element in the tabstrip
- add role="tablist" to the <kendo-tabstrip> element
- use tabindex + some styling to improve tabs focusing

**Comment by siyka-marinova** [comment]
Fix is available in the [latest dev. version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/#toc-obtaining-development-builds) of `@progress\kendo-angular-layout` package

**Comment by elena-gancheva** [comment]
The fix is available in the latest `4.1.5 version` of the `@progress/kendo-angular-layout` package.
